### ISSUE TITLE
Remove Apache 2.0 license from the Riseclipse OCL rule

### ISSUE THREAD

**Issue by Sander3003** [com-pas/compas-scl-validator#228]
**Subject of the issue**

busbar.OCL has 2 licences, this can only be one license. (https://github.com/com-pas/compas-scl-validator/commit/80e770fe31887c769f46ade81c9f58603cd24d67)

** Solution** 
Remove the Apache 2 license from the OCL rule and the place it with the existing licence.
If the build is broke due to a technical issues, ignore it

Add the EPL to the license to the Readme.md for this repository

**Comment by nicorikken** [comment]
Would it be possible to contact EDF to license it as Apache-2.0, in line with this project license?

**Comment by Sander3003** [comment]
Yes we can, Is their a big difference with the Eclipse Public License v2.0 license?
